tests/test_hash.py

```diff
@@ -50,7 +50,7 @@ def test_sha384(msg):
     assert(output.rstrip() == hashlib.sha384(msg.encode()).hexdigest())
 
 @helpers.filtered_test
-@pytest.mark.parametrize('msg', ['', 'a', 'abc', '1234567890123456789012345678901678901567890'])
+@pytest.mark.parametrize('msg', ['', 'a', 'abc', '1234567890123456789012345678901678901567890', '1234567890123456789012345678901678901567890andthensometohavemorethan64bytes', '1234567890123456789012345678901678901567890andlotsmoretexttosurelyandwithoutdoubtgobeyondthe128bytesrequiredtotriggerallincrementalblocklogiccases'])
```

Comment:
Were you able to confirm that these tests indeed fail with the old code?

Reply:
> Were you able to confirm that these tests indeed fail with the old code?

Yes